types/levelup/index.d.ts

```diff
@@ -108,7 +108,7 @@ declare namespace levelup {
         isOpen(): boolean;
         isClosed(): boolean;
 
-        readonly status: "new" | "opening" | "open" | "closing" | "closed";
+        readonly status: "closed" | "open" | "opening" | "new" | "closing";
```

Comment:
what was wrong with this one? it follows readme listing?
https://github.com/Level/levelup/blob/d31652d30a96b0132841bc4814b1f6a444f76e34/README.md#dbstatus

Reply:
As per [this comment](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61273#issuecomment-1185465580), it looks like `dtslint` had some issue, that it was failing for me either on TypeScript 4.7 or TypeScript 4.8 with different ordering (including the one that it's in `master` at the moment.

I assume that it has to be some `dtslint` bug fired by my changes, as it's passing fine on `master`. I included it here though, as otherwise it would not pass CI/CD, while it looks like the change is not harmful - it's just different order of literal union.

Reply:
ok, let's see how it works once merged (I'll revert this specific change later), thanks!